tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -152,7 +166,9 @@ public void constructor_with_maxLifeTime_should_initialize_instance()
             var subject = new ConnectionSettings(maxLifeTime: maxLifeTime);
 
             subject.ApplicationName.Should().BeNull();
-            subject.Authenticators.Should().Equal(__defaults.Authenticators);
+            subject.AuthenticatorFactories.Should().NotBeNull();
+            var authenticators = subject.AuthenticatorFactories.Select(a => a.Create());
+            authenticators.Should().BeEmpty();
```

Comment:
Replace 169-171 with:

            subject.AuthenticatorFactories.Should().BeEmpty();